apps/easy-messe/pages/offer-mass/index.tsx

```diff
@@ -87,14 +105,35 @@ export default function OrderMass() {
                     centered
                 >
                     {
-                        tabTitle.map((tabTitle, index) => (
+                        tabTitle.map(({ title, color, invisible, badgeContent }, index) => (
                             <Tab
                                 disableRipple
                                 key={index}
-                                label={tabTitle}
+                                label={
+                                    <Badge
+                                        invisible={invisible}
+                                        color={color}
+                                        badgeContent={badgeContent}
+                                    >
+                                        {title}
+                                    </Badge>
+                                }
                             />
                         ))
                     }
+                    {/* {
+                        tabTitle.map(({ title, badgeContent, color }, index) => (
+                            <Badge
+                                key={index}
+                                badgeContent={badgeContent}
+                            >
+                                <Tab
+                                    disableRipple
+                                    label={title}
+                                />
+                            </Badge>
+                        ))
+                    } */}
```

Comment:
remove commented code before requesting for a review.

commented code shouldn't go to prod